docs/pages/includes/database-access/multiple-instances-tip.mdx

```diff
@@ -0,0 +1,8 @@
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  A single Teleport process can run multiple services, for example
+  multiple Database Access instances as well as other services such the
+  SSH Service or Application Service.
+</Admonition>
```

Comment:
If we're going to create this partial we should use it everywhere. Here are some other pages I found that could use it:

docs/pages/database-access/guides/azure-sql-server-ad.mdx
docs/pages/database-access/guides/mysql-cloudsql.mdx
docs/pages/database-access/guides/postgres-cloudsql.mdx
docs/pages/includes/database-access/db-configure-start.mdx
docs/pages/includes/database-access/multiple-instances-tip.mdx

Reply:
Do you want to add this now or the subsequent prs for Azure and GCP helm installs?  The last 2 are includes so it would have to be a copy and paste

Reply:
Yes will do